docs/grid_model/network_subnetwork.md

```diff
@@ -819,9 +820,38 @@ A busbar section is a non impedant element used in a node/breaker substation top
 (internal-connection)=
 ## Internal connection
 
-**Internal connection**  
+[![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/VoltageLevel.NodeBreakerView.InternalConnection.html)<br>
 An internal connection is a non-impedant connection between two components in a voltage level.
 
 <!--
 <span style="color:red"> TODO</span>
--->
\ No newline at end of file
+-->
+
+(overload-management-system)=
+## Overload management systems
+
+[![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/OverloadManagementSystem.html)<br>
+An overload management system is an automation system that makes network actions when a current limit condition is met.
+
+Overload management systems are defined as a list of Trippings pairing one action and one limit.
```

Comment:
Do we have a previous definition of trippings? Maybe you should be more descriptive: what is a tripping and what tripping do we support?